rebar.config

```diff
@@ -1,39 +1,47 @@
 %% -*- erlang -*-
 {erl_opts, [
-            debug_info,
-            {parse_transform, lager_transform},
-            {i, "./_build/default/plugins/gpb/include"}
-           ]}.
+    debug_info,
+    {parse_transform, lager_transform},
+    {i, "./_build/default/plugins/gpb/include"}
+]}.
 
 {cover_export_enabled, true}.
 {cover_enabled, true}.
 
 {deps, [
-        {blockchain, {git, "https://github.com/helium/blockchain-core.git",
-                      {branch, "master"}}},
-        {hbbft, {git, "https://github.com/helium/erlang-hbbft.git",
-                 {branch, "master"}}},
-        {dkg, {git, "https://github.com/helium/erlang-dkg.git", {branch, "master"}}},
-        {ecc508, {git, "https://github.com/helium/ecc508.git", {branch, "master"}}},
-        {ebus, {git, "https://github.com/helium/ebus.git", {branch, "master"}}},
-        {jsx, {git, "https://github.com/talentdeficit/jsx.git", {branch, "v2.8.0"}}},
-        {kvc, {git, "https://github.com/etrepum/kvc", {tag, "v1.7.0"}}},
-        {longfi, {git, "https://github.com/helium/longfi-erlang", {tag, "0.2.0"}}},
-        recon
-       ]}.
+    {blockchain, {git, "https://github.com/helium/blockchain-core.git", {branch, "master"}}},
+    {hbbft, {git, "https://github.com/helium/erlang-hbbft.git", {branch, "master"}}},
+    {dkg, {git, "https://github.com/helium/erlang-dkg.git", {branch, "master"}}},
+    {ecc508, {git, "https://github.com/helium/ecc508.git", {branch, "master"}}},
+    {ebus, {git, "https://github.com/helium/ebus.git", {branch, "master"}}},
+    {jsx, {git, "https://github.com/talentdeficit/jsx.git", {branch, "v2.8.0"}}},
+    {kvc, {git, "https://github.com/etrepum/kvc", {tag, "v1.7.0"}}},
+    {longfi, {git, "https://github.com/helium/longfi-erlang", {tag, "0.2.0"}}},
+    recon
+]}.
 
 {xref_checks, [
-               undefined_function_calls,
-               undefined_functions,
-               %locals_not_used,
-               deprecated_function_calls,
-               deprecated_functions
-              ]}.
+    undefined_function_calls,
+    undefined_functions,
+    %locals_not_used,
+    deprecated_function_calls,
+    deprecated_functions
+]}.
+
+{project_plugins, [
+    erlfmt,
+    {rebar3_eqc, "1.3.0"}
+]}.
+
+{erlfmt, [
+    %% verbose,
+    write,
+    {print_width, 100}
```

Comment:
I'm not a fan of specifying this. If you want to format a single file with the `escript` version of `erlfmt`, e.g. via an editor command, this setting doesn't get picked up and the reformatted results may differ from what `rebar3 fmt` would do.

Reply:
(I admit this is a ridiculous state of affairs compared to say `clang-format` with its `some-repo/.clang-format` file)

Reply:
I guess my hope here was to have this be in a pre-commit hook or something like that, so we wouldn't have to specify it.  `rebar3 fmt` takes less than a second on my machine though, so maybe a single-file version isn't needed?

Reply:
I believe rebar3 fmt can also be told to crash if the file is not formatted, so we could make that part of CI